### PR TITLE
Torch.ravel Frontend 

### PR DESCRIPTION
**Pull request by nassimberrada**

### CODE REVIEW

**Comment by nassimberrada** [comment]
**Error**: Field elements must be 2- or 3-tuples
>           input=np.asarray(x, dtype=input_dtype),

File: ivy/functional/frontends/torch/miscellaneous_ops.py [hunk 1]
```diff
@@ -154,3 +154,8 @@ def logcumsumexp(input, dim, *, out=None):
 
 def repeat_interleave(input, repeats, dim=None, *, output_size=None):
     return ivy.repeat(input, repeats, axis=dim)
+
+
+def ravel(input):
+    reshape_shape = (int(ivy.prod(ivy.array(tuple(input.shape)))), 1)
+    return ivy.array(input, dtype=input.dtype).reshape(reshape_shape)
```
Comment (Ishticode) [inline ivy/functional/frontends/torch/miscellaneous_ops.py:161]: I see what you're done here. But there is a quicker way:
`ivy.reshape(input, (-1,))` should just return the flattened array any way.

**Review by Ishticode — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/torch/miscellaneous_ops.py (hunk 1 above)
Comment (nassimberrada) [reply ivy/functional/frontends/torch/miscellaneous_ops.py]: Yes that does make it much cleaner ! 
I made that change, i think the tests pass now

**Comment by nassimberrada** [comment]
**Status**: Tests Passed

**Review by Ishticode — APPROVED** [review]

**Comment by nassimberrada** [comment]
There was a slight issue in the testing inputs indeed. I ran the tests locally several times and it passes now

**Pull request merged**